src/02_datatypes.py

```diff
@@ -14,8 +14,13 @@
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
+y = 7;
+print(x + y)
```

Comment:
instead of redeclaring the variable each time, you can change the datatype in the same line for that print.

print(x +int(y))
print(str(x) + y)